Routing/Loader/Reader/RestActionReader.php

```diff
@@ -333,39 +333,41 @@ public function read(RestRouteCollection $collection, \ReflectionMethod $method,
 
             // add route to collection
             $route = new Route(
-                $path, $defaults, $requirements, $options, $host, [], $methods, $condition
+                $path, $defaults, $requirements, $options, $host, [], $methods, $versionCondition
             );
             $this->addRoute($collection, $routeName, $route, $isCollection, $isInflectable);
         }
     }
 
     /**
-     * Determine the Route condition by combining Route annotations with Version annotation.
-     *
-     * @param \ReflectionMethod $method
-     * @param RouteAnnotation   $annotation
+     * @return string|null
+     */
+    private function getVersionCondition()
+    {
+        if (empty($this->versions)) {
+            return;
+        }
+
+        return sprintf("request.attributes.get('version') in ['%s']", implode("','", $this->versions));
```

Comment:
Agree with the space after the comma to improve readability. But why the double quotes? Any reason apart from aesthetics? Just curious :)